lib/battle/CBattleInfoCallback.cpp

```diff
@@ -1303,8 +1303,8 @@ AttackableTiles CBattleInfoCallback::getPotentiallyAttackableHexes(const battle:
 
 				// if targeted double-wide creature is attacked from above or below ( -> second hex is also adjacent to attack origin)
 				// then dragon breath should target tile on the opposite side of targeted creature
-				if (BattleHex::mutualPosition(attackOriginHex, secondHex) != BattleHex::NONE)
-					nextHex = secondHex.cloneInDirection(direction, false);
+				/*if(BattleHex::mutualPosition(attackOriginHex, secondHex) != BattleHex::NONE)
```

Comment:
The most interesting part is this logic. If I uncomment it the client sees dragon breath hitting creatures which server live untouched.